omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -514,6 +515,20 @@ def generate_config(node_name)
         PrivateChef["default_listen_address"] = "::"
       end
 
+      # Transition Solr memory and JVM settings from OSC11 to Chef 12.
```

Comment:
I am +1 on doing things like this but it should be moved into a function of its own.  Perhaps named "handle_old_solr_config"